code/game/atoms.dm

```diff
@@ -793,15 +793,26 @@ GLOBAL_LIST_EMPTY(blood_splatter_icons)
 /atom/proc/clean_blood(radiation_clean = FALSE)
 	germ_level = 0
 	if(radiation_clean)
-		var/datum/component/radioactive/healthy_green_glow = GetComponent(/datum/component/radioactive)
-		if(!QDELETED(healthy_green_glow))
-			healthy_green_glow.strength = max(0, (healthy_green_glow.strength - (RAD_BACKGROUND_RADIATION * 2)))
-			if(healthy_green_glow.strength <= RAD_BACKGROUND_RADIATION)
-				qdel(healthy_green_glow)
+		clean_radiation(clean_factor = 2)
```

Comment:
```suggestion
		clean_radiation()
```
You dont need to override the default, or use the full named arg here

Reply:
I think I was trying to follow the style set by clean_blood there, where the arg seems to always be named. But if you want it changed, sure.